vraptor-javatime/src/main/java/br/com/caelum/vraptor/converter/InstantConverter.java

```diff
@@ -0,0 +1,44 @@
+package br.com.caelum.vraptor.converter;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+
+import br.com.caelum.vraptor.Convert;
+
+@RequestScoped
+@Convert(Instant.class)
+public class InstantConverter implements Converter<Instant> {
+	private final Logger log;
+	
+	/** @deprecated CDI */ @Deprecated
+	InstantConverter() { this(null); }
+	
+	@Inject
+	public InstantConverter(final Logger log) {
+		this.log = log;
+	}
+	
+	public Instant convert(final String value, final Class<? extends Instant> type) {
+		if (log.isDebugEnabled()) {
+			log.debug("Converting \"{}\" to Instant", value);
+		}

```

Comment:
this sounds great, but it's important to have all converters using the same conventions.
could you remove this log and fill a issue or another PR adding those logs on all of them?